providers/base/tests/test_pipewire_utils.py

```diff
@@ -20,11 +20,44 @@
 
 import unittest
 import sys
-from unittest.mock import MagicMock
+from unittest.mock import MagicMock, patch, call
 sys.modules["gi"] = MagicMock()
 sys.modules["gi.repository"] = MagicMock()
-from unittest.mock import patch
 from pipewire_utils import *
+from io import StringIO
+
+
+class RunCommandTests(unittest.TestCase):
+
+    @patch("subprocess.check_output")
```

Comment:
This won't be necessary if we just add the compare_status function to pipewire_utils.py